x-pack/plugins/cases/public/containers/use_get_cases.test.tsx

```diff
@@ -32,11 +31,14 @@ describe.skip('useGetCases', () => {
 
   it('calls getCases with correct arguments', async () => {
     const spyOnGetCases = jest.spyOn(api, 'getCases');
-    const { waitForNextUpdate } = renderHook(() => useGetCases(), {
+    const { waitFor } = renderHook(() => useGetCases(), {
       wrapper: appMockRender.AppWrapper,
     });
 
-    await waitForNextUpdate();
+    await waitFor(() => {
+      expect(spyOnGetCases).toBeCalled();
```

Comment:
nitpick: would it be better to waitFor the result of this function? I'm assuming `useGetCases` is async.

But if this passes then I guess it's fine too haha.

Reply:
Good point. I did this because sometimes you get a timeout message without further explanation if the test is flaky. By having the result of the function outside, we will get a message about what properties of the object are different. I do not generally do it or suggest it but for this test I did it in case it will be flaky again. Do you want me to change it back?

Reply:
No its all good